doc/source/advanced.rst

```diff
@@ -473,7 +474,14 @@ The ``runtime_env`` is a Python dictionary including one or more of the followin
 
   Note: Setting this option per-task or per-actor is currently unsupported.
 
-- ``pip`` (List[str] | str): Either a list of pip packages, or a string containing the path to a pip 
+  Note: If your working directory contains a `.gitignore` file, the files and paths specified therein will not be uploaded to the cluster.
+
+- ``excludes`` (List[str]): When used with ``working_dir``, specifies a list of files or paths to exclude from being uploaded to the cluster.
+  This field also supports the pattern-matching syntax used by ``.gitignore`` files: see `<https://git-scm.com/docs/gitignore>`_ for details.
+
+  - Example: ``["my_file.txt", "path/to/dir", "*.log"]``
```

Comment:
@iycheng When you get a chance, can you let me know if this is incorrect or if I missed anything here?